tests/h/models/document_test.py

```diff
@@ -747,6 +748,15 @@ def err():
         with pytest.raises(ConcurrentUpdateError):
             document.merge_documents(db_session, merge_data)
 
+    def test_merge_documents_logs_when_its_called(self, caplog, db_session, merge_data):
+        caplog.set_level(logging.INFO)
+
+        document.merge_documents(db_session, merge_data)
+
+        assert caplog.record_tuples == [
+            ("h.models.document", 20, "Merging 3 documents")
+        ]
+
```

Comment:
I wouldn't test this. That way lies madness I think

Reply:
Logging is a function of the code like any other, that we're going to rely on to monitor this. It's worth testing because we do want to know that it works. For example if I want to change the logging to use `"Merging %s documents"` I can verify that it works by re-running this test, rather than going through a complicated series of steps (involving wiping my DB and annotating three documents in the right order) to trigger a `merge_documents()` and see the log message in my terminal